<launch>
<node pkg="tf" type="static_transform_publisher" name="right_up_corner" args="10 10 0 0 0 0  Pioneer3AT/odom /obstacle/right_up_corner 200" />
<node pkg="tf" type="static_transform_publisher" name="left_up_corner" args="10 -10 0 0 0 0  Pioneer3AT/odom /obstacle/left_up_corner 200" />
<node pkg="tf" type="static_transform_publisher" name="right_corner" args="-10 10 0 0 0 0  Pioneer3AT/odom /obstacle/right_corner 200" />
<node pkg="tf" type="static_transform_publisher" name="left_corner" args="-10 -10 0 0 0 0 Pioneer3AT/odom /obstacle/left_corner 200" />

<node pkg="tf" type="static_transform_publisher" name="middle" args="5 0 0 0 0 0  Pioneer3AT/odom /obstacle/middle 100" />
</launch>
